### PR TITLE
feat(ui, server): enable admin to generate user reset password links

### PR DESCRIPTION
**Pull request by liangfung**
## Screen record
https://jam.dev/c/a7e9cf6e-344c-4e8d-89c7-cae2e2d8f8a3

### CODE REVIEW

File: ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx [hunk 1]
```diff
@@ -243,3 +223,127 @@ export default function UsersTable() {
...
+            className="cursor-pointer gap-1"
+          >
+            <span className="ml-2">Initate resetting password</span>
+          </DropdownMenuItem>
+        </DropdownMenuContent>
+      </DropdownMenu>
+      <AlertDialog open={open} onOpenChange={onOpenChange}>
+        <AlertDialogContent>
+          <AlertDialogHeader>
+            <AlertDialogTitle>Initiate resetting password</AlertDialogTitle>
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx:325]:
```suggestion
            <AlertDialogTitle>Reset password</AlertDialogTitle>
```

File: ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx [hunk 2]
```diff
@@ -243,3 +223,127 @@ export default function UsersTable() {
...
+              className="cursor-pointer"
+            >
+              <span className="ml-2">Activate</span>
+            </DropdownMenuItem>
+          )}
+          <DropdownMenuItem
+            onSelect={() => setOpen(true)}
+            className="cursor-pointer gap-1"
+          >
+            <span className="ml-2">Initate resetting password</span>
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx:318]:
```suggestion
            <span className="ml-2">reset password</span>
```

File: ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx [hunk 3]
```diff
@@ -243,3 +223,127 @@ export default function UsersTable() {
...
+          </DropdownMenuItem>
+        </DropdownMenuContent>
+      </DropdownMenu>
+      <AlertDialog open={open} onOpenChange={onOpenChange}>
+        <AlertDialogContent>
+          <AlertDialogHeader>
+            <AlertDialogTitle>Initiate resetting password</AlertDialogTitle>
+            <AlertDialogDescription>
+              Are you sure you want to create a password reset link for{' '}
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx:327]: By clicking "Yes," a password reset link will be generated for <span>...</span>. The password won't be modified until the user follows the instructions in the link to make the change.

**Review by wsxiaoys — APPROVED** [review]
Otherwise LGTM

File: ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx (hunk 3 above)
Comment (liangfung) [reply ee/tabby-ui/app/(dashboard)/settings/team/components/user-table.tsx]: image

**Pull request merged**